test/services/helpers/services/courseGroups.js

```diff
@@ -0,0 +1,41 @@
+const { courseModel } = require('../../../../src/services/user-group/model');
+
+let createdCourseIds = [];
+
+const removeManyCourses = (ids) => courseModel.deleteMany({ _id: { $in: ids } }).lean().exec();
+
+const createTestCourseGroup = (app, opt) => ({
+	// required fields for base group
+	name = 'testCourse',
+	schoolId = opt.schoolId,
+	userIds = [],
+	classIds = [],
+	courseId = undefined,
+} = {}) => app.service('courseGroups').create({
+	// required fields for user
+	name,
+	schoolId,
+	userIds,
+	classIds,
+	courseId,
+}).then((course) => {
+	createdCourseIds.push(course._id.toString());
+	return course;
+});
+
+const cleanup = () => {
+	if (createdCourseIds.length === 0) {
+		return Promise.resolve();
+	}
+	const ids = createdCourseIds;
+	createdCourseIds = [];
+	return removeManyCourses(ids);
+};
+
+module.exports = (app, opt) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; parenthesize the returned value and move it immediately after the `=>`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5387400)